sdk-base/src/main/java/com/mapbox/maps/plugin/locationcomponent/IsPointOnLocationPuckListener.kt

```diff
@@ -0,0 +1,13 @@
+package com.mapbox.maps.plugin.locationcomponent
+
+/**
+ * Listener that gets invoked when the is point on location puck validation finished.
+ */
+fun interface IsPointOnLocationPuckListener {
+  /**
+   * This method is called when the is point on location puck validation finished.
+   *
+   * @param isOnLocationPuck true if the given point is on the location puck, false otherwise.
+   */
+  fun isPointOnLocationPuck(isOnLocationPuck: Boolean)
```

Comment:
I guess it is always called on non-UI thread, we could add `@WorkerThread` annotation then.

Reply:
I guess we can not make that assumption, since this is just an utility function and could be used anywhere?